src/java/org/apache/cassandra/cql3/statements/schema/CreateKeyspaceStatement.java

```diff
@@ -56,6 +62,10 @@ public CreateKeyspaceStatement(String keyspaceName, KeyspaceAttributes attrs, bo
 
     public Keyspaces apply(Keyspaces schema)
     {
+        if (ClientWarn.instance.get() == null)
+            ClientWarn.instance.captureWarnings();
+        int previousWarningsSize = ClientWarn.instance.getWarnings() == null ? 0 : ClientWarn.instance.getWarnings().size();
```

Comment:
`numWarnings()` could be used here

Reply:
Will resolve depending on how we resolve the above. Resolving for now to keep this clear-ish

Reply:
I am -1 bc it would break the semantics of a non-inited `ClientWarn` instance being non functional until now. If we do that change some methods will fail and others will work when not the instance is not inited. In any case I have no hard opinion here. I can do it if you guys prefer it.

Reply:
My previous comment in AlterKeyspaceStatement stays - let's then change a bit the naming. Thanks :-)

Reply:
Added `numWarnings()`